push_processor/tests/test_handler.py

```diff
@@ -65,6 +65,24 @@ def test_lambda(self):
         ), None)
         eq_(result, None)
 
+    def test_lambda_with_json(self):
+        import push_processor.handler as handler
+        handler.settings = {
+            "s3_bucket": "push-test",
+            "s3_key": "opts.js",
+            "redis_port": 6379,
+            "db_tablename": "push_messages_db",
+            "file_type": "json"
+        }
+
+        result = handler.aws_lambda(dict(
+            Records=[dict(s3=dict(
+                          bucket=dict(name="push-test"),
+                          object=dict(key="push_dash_logs.json")
+                          ))]
+        ), None)
+        eq_(result, None)
```

Comment:
Can you mock process_json_stream so you can verify that it gets called? Otherwise, not really sure what you're testing here other than doing code coverage.